platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileManager.java

```diff
@@ -457,100 +220,10 @@ private Path getSignedStateDir(final long round) {
-            return RECONNECT;
-        }
-
-        final StateConfig stateConfig = configuration.getConfigData(StateConfig.class);
-        final int saveStatePeriod = stateConfig.saveStatePeriod();
-        if (saveStatePeriod <= 0) {
-            // periodic state saving is disabled
-            return null;
-        }
-
-        // FUTURE WORK: writing genesis state to disk is currently disabled if the saveStatePeriod is 0.
-        // This is for testing purposes, to have a method of disabling state saving for tests.
-        // Once a feature to disable all state saving has been added, this block should be moved in front of the
-        // saveStatePeriod <=0 block, so that saveStatePeriod doesn't impact the saving of genesis state.
-        if (previousTimestamp == null) {
-            // the first round should be saved
-            return FIRST_ROUND_AFTER_GENESIS;
-        }
-
-        if ((signedState.getConsensusTimestamp().getEpochSecond() / saveStatePeriod)
-                > (previousTimestamp.getEpochSecond() / saveStatePeriod)) {
-            return PERIODIC_SNAPSHOT;
-        } else {
-            // the period hasn't yet elapsed
-            return null;
-        }
-    }
-
-    /**
-     * Determine if a signed state should eventually be written to disk. If the state should eventually be written, the
-     * state's {@link SignedState#markAsStateToSave} method will be called, to indicate the reason
-     *
-     * @param signedState the signed state in question
-     * @param source      the source of the signed state
-     */
-    public synchronized void determineIfStateShouldBeSaved(
-            @NonNull final SignedState signedState, @NonNull final SourceOfSignedState source) {
-
-        final StateToDiskReason reason = shouldSaveToDisk(signedState, previousSavedStateTimestamp, source);
-
-        // if a null reason is returned, then there isn't anything to do, since the state shouldn't be saved
-        if (reason == null) {
-            return;
-        }
-
-        logger.info(
-                STATE_TO_DISK.getMarker(),
-                "Signed state from round {} created, "
-                        + "will eventually be written to disk once sufficient signatures are collected, for reason: {}",
-                signedState.getRound(),
-                reason);
-
-        previousSavedStateTimestamp = signedState.getConsensusTimestamp();
-        signedState.markAsStateToSave(reason);
-    }
-
-    /**
-     * This should be called at boot time when a signed state is read from the disk.
-     *
-     * @param signedState the signed state that was read from file at boot time
-     */
-    public synchronized void registerSignedStateFromDisk(final SignedState signedState) {
-        previousSavedStateTimestamp = signedState.getConsensusTimestamp();
-    }
-
     /**
      * Purge old states on the disk.
      */
-    private synchronized void deleteOldStates() {
+    private long deleteOldStates() {
```

Comment:
Could you write a brief doc explaining what the returned `long` represents?

Reply:
done